doc/DOCUMENTATION.org

```diff
@@ -476,15 +476,15 @@ layers are not source controlled/.
 *** Using an external Git repository
 This is the recommended way to manage your private layers.
 
-The best approach is to store all your private layers into an external Git
-repository. It is especially a good practice to store them in your =dotfiles=
-repository if you have one. Put also your =~/.spacemacs= file in it.
+The best approach is to store all your private layers in an external Git
+repository. It is good practice to store them in your =dotfiles=
```

Comment:
Not a native english speaker but I find it odd to say `It is good practice` without an article. Should it be `It is a good practice`?

Reply:
I think you're right. I've updated the structure of the sentence to something which might read better.